src/frontend/src/lib/components/ui/Hr.svelte

```diff
@@ -1 +1,9 @@
-<hr class="bg-dark-blue opacity-10" style="width: 100%; border: 0.05rem solid" />
+<script lang="ts">
+	export let spacing: 'none' | 'lg' = 'none';
+</script>
+
+<hr
+	class="bg-dark-blue opacity-10"
+	class:my-6={spacing === 'lg'}
```

Comment:
I dont't know if it makes sense to you, but we have `Size` type now

`type Size = 'xs' | 'sm' | 'md' | 'lg' | 'xl';`

so you could use `Pick<Size, 'lg'>` if you want

(though it is not globally exported yet...)

Reply:
`Pick<Size, 'lg'>` is not possible though because `Size` is a `type` or I am missing something?

Reply:
right! sorry! maybe `Extract<Size, 'lg'>`? But for just one, maybe not worth it? boh

Reply:
Nice, I did not knew about `Extract`! Done. https://github.com/dfinity/oisy-wallet/pull/3181/commits/88382d8c51d25b0d6ceb8e32b40f3f3b6aea66e5